MapboxDirections/MBRouteOptions.swift

```diff
@@ -119,9 +155,21 @@ open class RouteOptions: DirectionsOptions {
             URLQueryItem(name: "alternatives", value: String(includesAlternativeRoutes)),
             URLQueryItem(name: "continue_straight", value: String(!allowsUTurnAtWaypoint))
         ])
-
-        if includesExitRoundaboutManeuver {
-            queryItems.append(URLQueryItem(name: "roundabout_exits", value: String(includesExitRoundaboutManeuver)))
+        
+        switch profileIdentifier {
+        case .automobile, .automobileAvoidingTraffic:
```

Comment:
These switch statements might become more invasive over time. Maybe we can extend the MBDirectionsProfileIdentifier enumeration so that a given profile knows which parameters it supports?